src/main/java/seedu/address/model/book/Book.java

```diff
@@ -142,7 +142,7 @@ public Book returnBook() {
     public Book addToLoanHistory(Loan loan) {
         LoanList newHistory = new LoanList();
         newHistory.add(loan);
-        this.getLoanHistory().forEach(currentLoan -> newHistory.add(currentLoan));
```

Comment:
Initially used this notation to make it more readable but I guess it's up to personal preference! :)